torch/csrc/generic/methods/SparseTensor.cwrap

```diff
@@ -157,6 +155,37 @@ PyObject * THSPTensor_(size)(PyObject *self, PyObject *args, PyObject *kwargs)
 ]]
 
 [[
+  name: t
+  sparse: yes
+  with_stateless: True
+  cname: newTranspose
+  return: THSTensor*
+  before_call: |
+    long nDimI = ((THSPTensor*)${arg0})->cdata->nDimensionI;
+    long nDimV = ((THSPTensor*)${arg0})->cdata->nDimensionV;
+    THPUtils_assert(nDimI == 2 && nDimV == 0, "t() expects a 2D sparse tensor, but self is %ldD indices and %ldD values", nDimI, nDimV);
```

Comment:
Is `transpose` supported for hybrid tensors? If yes, why not only check `nDimI + nDimV == 2`? If no, don't we need another check in the `transpose` method?